test-suite/Makefile

```diff
@@ -118,11 +118,14 @@ bugs: $(BUGS)
 
 clean:
 	rm -f trace .lia.cache output/MExtraction.out
-	$(SHOW) "RM        <**/*.vo> <**/*.vio> <**/*.log> <**/*.glob>"
+	$(SHOW) "RM        <**/*.stamp> <**/*.vo> <**/*.vio> <**/*.log> <**/*.cache> <**/*.glob>"
 	$(HIDE)find . \( \
-	  -name '*.vo' -o -name '*.vio' -o -name '*.log' -o -name '*.glob' \
+	  -name '*.stamp' -o -name '*.vo' -o -name '*.vio' -o -name '*.log'  -o -name '*.cache' -o -name '*.glob' \
```

Comment:
Could it be because of this change?

Reply:
>I will remove "*.cache" here, but this can't be the cause, since this is only the 'clean'. And the cache was present when I started the tests.

"make test-suite" from the main Makefile cleans the test suite before running it.
https://github.com/coq/coq/blob/97ee8fbd0bf917c29e47f746c0a28623ebc7da9a/Makefile.build#L550-L552
This is the way it's called in travis, so travis always deletes the cache in your PR.

Reply:
I'm seeing this error now:

```
Error:
while loading
C:\Users\Jim.DESKTOP-MMPNVGH\Documents\coqSource\coq\plugins\ltac\ltac_plugin.cmxs:
implementation mismatch on Vernacentries
```

Is this caused by my recompiling the .ml (make coqbinaries) without recompiling the .v files (which is so slow)?

Reply:
Try `make pluginsopt`.